mangaki/mangaki/models.py

```diff
@@ -355,7 +388,8 @@ def __str__(self):
 
 
 class Tag(models.Model):
-    title = models.TextField(unique = True)
+    title = models.CharField(max_length=255)
+    anidb_tag_id = models.IntegerField(default=0, unique=True)
```

Comment:
I had tried to leave that field with no `default=0` nor `null=True` attributes, however, Django seems to dislike having no default value for a non-nullable field. That's why I added a default value. Here's what Django says when I try to `makemigrations` :
```
You are trying to add a non-nullable field 'anidb_tag_id' to tag without a default; we can't do that (the database needs something to populate existing rows).
Please select a fix:
 1) Provide a one-off default now (will be set on all existing rows with a null value for this column)
 2) Quit, and let me add a default in models.py
```